### DISCUSSION TITLE
I can't set a screensaver despite installing the extension

### DISCUSSION THREAD

**Discussion by elygie** [rocksdanister/lively#2343] [Q&A] [answered]
I installed the extension to configure my screensaver.
After several checks and restarts of the computer, nothing works, it always tells me to install the extension.

What should I do? Do you have a solution?

Thanks in advance ;(

**Comment by ToniEliamokhtar** [comment]
I have the same problem, please tell me how you solved it, if you did...

**Comment by rocksdanister** [comment]
Make sure the .scr file is copied to `C:\Windows` https://github.com/rocksdanister/lively/issues/2385

Run the `lively_screensaver_checker` and post screenshot:
https://github.com/rocksdanister/lively-beta/releases

**Comment by h3xu588** [comment]
No, all .scr files are located in `C:\Windows\System32`. Just extract the zip and make sure that the .scr file is located there.

**Reply by rocksdanister** [reply]
No, I am checking the file in `C:\Windows`

**Comment by rocksdanister** [comment, marked as answer]
> The message can be dismissed in the next update.

https://github.com/rocksdanister/lively/issues/2385#issuecomment-2598283103
